.github/workflows/rust.yml

```diff
@@ -98,3 +97,26 @@ jobs:
         run: cargo build
       - name: cargo test
         run: cargo test
+  test-rpm-build:
```

Comment:
I'd lean toward putting this in a separate workflow, since it's unrelated to the other jobs in this file.